docs/sidebars.js

```diff
@@ -17,409 +17,349 @@ const isProd = process.env.NODE_ENV === 'production';
-                    label: 'Build on Composable: Ecosystem Development',
-                    link: {
-                        type: `doc`,
-                        id: `ecosystem/build-on-composable-ecosystem-development`,
-                    },
-                    collapsible: false,
-                    items: [
-                        'ecosystem/rfp-canonical-stablecoin-design-and-integration',
-                        'ecosystem/rfp-explorer',
-                    ]
-                },
-                'ecosystem/composable-grants',
-                'ecosystem/press-kit',
-                'ecosystem/the-composable-team',
+              "ecosystem/rfp-canonical-stablecoin-design-and-integration",
+              "ecosystem/rfp-explorer",
             ],
+          },
+          "ecosystem/composable-grants",
+          "ecosystem/press-kit",
+          "ecosystem/the-composable-team",
+        ],
+      },
+      {
+        type: "doc",
+        label: "Audits, Fixes & Bug Bounties",
+        id: "audits/audit-results-recommendations-and-remediations",
+      },
+      {
+        type: "doc",
+        label: "FAQs",
+        id: "faqs/faqs",
+      },
+  
+      {
+        type: "category",
+        label: "Legal Disclaimers and Disclosures",
+        collapsible: true,
+        collapsed: true,
+        items: [
+          "faqs/disclaimers-disclosures-for-composable-tokens",
+          "faqs/risk-factors",
+          "faqs/terms-of-use",
+        ],
+      },
+    ],
+    networks: [
+      {
+        type: "category",
+        label: "Picasso",
+        link: {
+          type: "doc",
+          id: "parachains/picasso-parachain-overview",
         },
-        {
-            type: 'doc',
-            label: 'Audits, Fixes & Bug Bounties',
-            id: 'audits/audit-results-recommendations-and-remediations',
-
+        collapsible: false,
+        collapsed: false,
+        items: [
+          "parachains/picasso/crowdloan",
+          "parachains/picasso/governance",
+          "parachains/picasso/token-transparency",
+          "parachains/picasso/pica-use-cases",
+        ],
+      },
+      {
+        type: "category",
+        label: "Composable",
+        link: {
+          type: "doc",
+          id: "parachains/composable-parachain-overview",
         },
-        {
-            type: 'doc',
-            label: 'FAQs',
-            id: 'faqs/faqs',
-
+        collapsible: false,
+        collapsed: false,
+        items: [
+          "parachains/composable/composable-crowdloan",
+          "parachains/composable/LAYR-tokenomics",
+          "parachains/composable/composable-council",
+        ],
+      },
+  
+      "parachains/centauri-chain",
+    ],
+  
+    centauri: [
+      "products/centauri/Dotsama-ibc",
+      "products/centauri/near-ibc-bridge",
+      "products/centauri/hyperspace-relayer",
+      "products/centauri/light-clients",
+      "products/centauri/merkle-mountain-ranges",
```

Comment:
MMR + light clients + beefy ibc light client etc can all be under architecture within the first category